src/gowrapper/ckks/params.go

```diff
@@ -63,6 +64,37 @@ func lattigo_getDefaultPQParams(paramEnum uint8) Handle6 {
 	return marshal.CrossLangObjMap.Add(unsafe.Pointer(&params))
 }
 
+//export lattigo_newParameters
+func lattigo_newParameters(logN uint64, qi *C.constULong, numQi uint8, pi *C.constULong, numPi uint8, logScale uint8) Handle6 {
```

Comment:
Why use `uint8` for `numQi`? `qi.size()` type is `size_t`, which has 8 bytes.
```
const vector<uint64_t> &qi
```

Reply:
Since this function is called across the language boundary, I want to be explicit with bit sizes so that there aren't any errors resulting from "C thinks type X has 16 bits, but Go thinks it has 32 bits". 8 bits is sufficient for this field since no valid, realistic parameter set has >255 primes.

I will add a check in the C code, but it will also just directly exit the program.